scripts/covid19_india/cases_count_districts_data/README.md

```diff
@@ -0,0 +1,39 @@
+# COVID-19 Cases Counts as reported by Ministry of Health and Family Welfare (MoHFW)
+
+
+## About the Dataset
+
+### Download URL
+You can download the raw csv with district-wise daily case counts data for India from [covid19india api](	https://api.covid19india.org/csv/latest/districts.csv)
+
+
+
+### Overview
+[MoHFW](https://www.mohfw.gov.in/) publishes cumulative confirmed, cured, death due to COVID-19 counts, everyday. It's published for every State and Union Territory. Sometimes it's published more than once a day. It's scraped by the [covid19india](https://covid19india.org).
+
+
+#### Cleaned Data
+- [COVID19_cases_indian_states.csv](COVID19_cases_indian_states.csv).
+
+#### Template MCFs
+- [COVID19_cases_indian_states.tmcf](COVID19_cases_indian_states.tmcf).
+
+
```

Comment:
should this part talk about district data/files?

Also, please indicate how we plan to resolve the places. using indianCensusAreaCode2011 property?